src/app/manager-dashboard/manager-dashboard.component.ts

```diff
@@ -84,4 +91,18 @@ export class ManagerDashboardComponent implements OnInit {
       }, error => this.planetMessageService.showAlert('An error occurred please try again.'));
   }
 
+  checkRequestAcceptedOrNot() {
+    this.couchService.allDocs('communityregistrationrequests', { domain: this.userService.getConfig().parentDomain }) .subscribe(data => {
```

Comment:
why you are looking after alldocs when you just need configuration with the code of current configuration
Also later we will have configuration synced back to local config so it will not require to request with remote parent